howtos/oem-guide.rst

```diff
@@ -0,0 +1,135 @@
+Continuing the freedoms originally granted, for folks to know what code their systems are running.
+
+Note that `The GPL Is a License, not a Contract <https://lwn.net/Articles/61292/>`_.
+
+`The Rockstor Project <https://opencollective.com/the-rockstor-project>`_
+is an `Open Collective <https://opencollective.com/>`_
+Non-Profit/Non-Business Open Source community endeavour.
+
+- All rockstor distributed code is FSF Free/Libre & OSI approved and available on our
+  `GitHub Organisation <https://github.com/rockstor>`_ site.
+
+- All 'rockstor' rpm packages are source (--format sdist) in-nature: i.e. non binary.
+
+- All released packages are build directly from publicly tagged GitHub releases,
+  for details see: `rockstor-rpmbuild <https://github.com/rockstor/rockstor-rpmbuild>`_.
+
+- All installers are built using an open source project and configuration,
+  for details see: `rockstor-installer <https://github.com/rockstor/rockstor-installer>`_.
+
+Our continued existence, i.e. ongoing-development & distribution services,
+depend entirely upon good-will and active community participation in our Open Collective membership Tiers.
+As such circumvention of these service orientated tiers (curated release channels / support / membership / etc),
+in any way, would result in no future communication or service to the offending agent.
+We protect, by way of our licensing choices, and consequent code availability, the end-users freedoms.
+That does not oblige any member of The Rockstor Project's team in any way: regardless of Tier membership.
+
+**Given the above freedoms, all-around, we can only request the following:**
+
+1. OEM hardware distributors/redistributors resellers honour our existing sustainability model.
+I.e. they have an appropriate number, given anticipated development team contacts per-year,
+of 'OEM Pre-installer subscription' memberships.
+Rockstor's continued development, and existence,
+depends entirely on interested parties appreciating both past and ongoing development team efforts and associated costs.
+
+The Rockstor Project development team, in-turn,
+will endeavour to prioritise OEM/reseller technical enquiries/requests as resources allow.
+I.e. a pre-installer's enquiries/concerns are assumed to relate to more than one install,
+where-as our non OEM contribution tiers relate to specific installs; i.e:
+
+- a. Stable Updates subscription €24/year.
+- b. Incident-based Support subscription (`Planned <https://rockstor.com/paid_support.html>`_) €240/incident/year.
+- c. OEM Pre-installer subscription (proposed) €960/year.
+
+.. note::
+
+    Each of the above Tiers is independent,
+    (a.) no implied support,
+    (b.) superset of (a.) including a single support incident.
+    (c.) OEM/pre-installer/for-profit entity membership: no implied end-user support.
+
+2. Any pre-install arrangement must honour our default of neither :ref:`update_channels` channel being enabled.
```

Comment:
2. Any pre-install arrangement must honour our default of neither :ref:`update_channels` channel being enabled, making it the end-users choice, which one to pursue, including none.